quickstart.sh

```diff
@@ -39,7 +39,7 @@ mkdir -p  $backupDir
 if [ -n "${dockerPSOutput}" ];
 then
     echo " ### Seems like a TestRail instance is already running"
-    echo "   To shut it down run 'docker-compose down -v' in this folder and then call this script again."
+    echo "   To shut it down run 'docker-compose/docker compose down -v' in this folder and then call this script again."
```

Comment:
@acuanico-tr-galt These doesn't seem related to chrome-php updates. Is this for your local only?

Reply:
@parboleda-tr-galt I included these updates to support newer version of docker compose (Compose v2).

Reply:
Let's separate this docker-compose changes in another PR